[Bug] [connector][mongodb] 缺少依赖包

### Search before asking

- [X] I had searched in the [issues](https://github.com/DTStack/chunjun/issues) and found no similar issues.


### What happened

当在yarn-pre-job sql模式执行从mongodb导出数据到其它数据源时找不到依赖异常。

```
Caused by: java.lang.NoClassDefFoundError: com/mongodb/AuthenticationMechanism
	at com.dtstack.chunjun.connector.mongodb.config.MongoClientConfig.<init>(MongoClientConfig.java:39)
	at com.dtstack.chunjun.connector.mongodb.table.MongodbDynamicTableFactory.createDynamicTableSource(MongodbDynamicTableFactory.java:66)
	at org.apache.flink.table.factories.FactoryUtil.createDynamicTableSource(FactoryUtil.java:163)
	... 58 more
Caused by: java.lang.ClassNotFoundException: com.mongodb.AuthenticationMechanism
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

### What you expected to happen

--

### How to reproduce

--

### Anything else

_No response_

### Version

master

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/chunjun-connectors/chunjun-connector-mongodb/pom.xml
+++ b/chunjun-connectors/chunjun-connector-mongodb/pom.xml
@@ -56,6 +56,10 @@
 		<plugins>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-antrun-plugin</artifactId>
 			</plugin>
 		</plugins>

